environments/prod/shutter.tfvars

```diff
@@ -0,0 +1,14 @@
+# Common variables - do not change these
+
+env                 = "prod"
```

Comment:
you don't need this it should all be setup automatically.

you do need to add your app to https://github.com/hmcts/azure-shutter-pages/blob/master/azure-pipelines.yml#L93-L95

make sure you choose SHAREDSERVICES, and if you want a non default shutter page then copy the default and make changes to it